packages/teams-js/src/public/geoLocation.ts

```diff
@@ -62,7 +64,7 @@ export namespace geoLocation {
     }
     const permissions: DevicePermission = DevicePermission.GeoLocation;
 
-    return new Promise<boolean>((resolve) => {
+    return new Promise<boolean>(resolve => {
```

Comment:
my eslint prettier is throwing error if I don't make this change. @erinha is this okay?